docs/deep-dive/03-dd-expose-function-serverless.md

```diff
@@ -80,25 +142,27 @@ Follows these steps:
 
```

Comment:
line 67: APIRule ?

Correct in other places in this document as well, if needed.